blocks/name.js

```diff
@@ -114,7 +114,7 @@ module.exports.resolve = ({serverless, variableUtils, slsHelper, logUtils}) => {
   productBucket = new Promise(async (resolve, reject) => {
     try {
       const accountDomainName = await resolveVariable('stencil(account):domain');
-      const bucketName = await getBucketName(accountDomainName);
+      const bucketName =  (await getBucketName(accountDomainName)).toLowerCase();
```

Comment:
Almost.
I'd prefer that returned value from `getBucketName()` call is already compliant to the bucket name requirements.